doc/sailfish-webview-webengine.qdoc

```diff
@@ -0,0 +1,159 @@
+/****************************************************************************
+**
+** Copyright (c) 2021 Open Mobile Platform LLC
+**
+****************************************************************************/
+
+/*!
+  \qmltype WebEngine
+  \inherits Item
+  \inqmlmodule Sailfish.WebEngine
+  \ingroup sailfishos-webview
+  \brief Provides access to the Web engine context
+
+  The \c{WebEngine} component is a singleton which provides access to the Web engine context.
+
+  The \c{WebEngine} QML type is exposed through the \c{Sailfish.WebEngine} QML
+  import namespace.
+
+  Exaple of use:
+
+  \code
+  import QtQuick 2.0
+  import Sailfish.Silica 1.0
+  import Sailfish.WebView 1.0
+  import Sailfish.WebEngine 1.0
+
+  ApplicationWindow {
+      id: root
+      initialPage: Component {
+          Page {
+              id: page
+              WebView {
+                  anchors.fill: parent
+                  active: true
+                  url: "http://www.sailfishos.org"
+              }
+
+              Component.onCompleted: {
+                  WebEngine.addObserver("clipboard:setdata")
+                  WebEngine.onRecvObserve.connect(function(message, data) {
+                      if (message == "clipboard:setdata" && !data.private) {
+                          console.log("Clipboard contents: " + data.data);
+                      }
+                  })
+              }
+          }
+      }
+  }
+  \endcode
+
+  \sa WebVew
+*/
+
+/*!
+  \qmlmethod bool WebEngine::isInitialized()
+  \brief Returns true if the component has been initialized, false otherwise.
+*/
+
+/*!
+  \qmlmethod bool WebEngine::isAccelerated()
+  \brief Returns true if the view is using graphics acceleration, false otherwise.
+
+  Allows the status of graphics acceleration to be queried.
+
+  Graphics acceleration may be Open GL or GLES, for example.
+*/
+
+/*!
+  \qmlmethod bool WebEngine::setIsAccelerated(aIsAccelerated)
+  \brief TBC
+*/
+
+/*!
+  \qmlmethod bool WebEngine::addComponentManifest(manifestPath)
+  \brief TBC
+*/
+
+/*!
+  \qmlmethod bool WebEngine::addObserver(aTopic)
+  \brief TBC
+*/
+
+/*!
+  \qmlmethod bool WebEngine::notifyObservers(topic, value)
+  \brief TBC
+*/
+
+/*!
+  \qmlmethod bool WebEngine::runEmbedding(aDelay)
```

Comment:
This is is the most crucial methods in order to start the engine.

Reply:
Actually, I must admit to being a bit confused about `runEmbedding()`. It's called in `SailfishOS::WebEngine::initialize()`, which is itself called in both `SailfishOSWebViewPlugin::initializeEngine()` and `SailfishOSWebEnginePlugin::initializeEngine()`, which will be run automatically in QML. So there should be no need for a developer to call it directly via QML I think.

So I think it should only appear in the C++ docs, for developers rolling their own initialisation. Does that seem fair?